apps/emqx_bridge_hstreamdb/src/emqx_bridge_hstreamdb_connector.erl

```diff
@@ -44,6 +45,8 @@
 
 %% -------------------------------------------------------------------------------------------------
 %% resource callback
+resource_type() -> hstreamsdb.
```

Comment:
```suggestion
resource_type() -> hstream.
```

Reply:
https://github.com/emqx/emqx/blob/master/apps/emqx_bridge/src/schema/emqx_bridge_enterprise.erl#L105
We utilize `hstreamdb` in the API; keep them consistent.